components/docs-chef-io/content/automate/chef_automate_profile.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "User Profile"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "User Profile"
+    parent = "automate/settings"
+    identifier = "automate/settings/chef_automate_profile.md User Profile"
+    weight = 120
++++
+
+## Overview
+
+Chef Automate users can manage their own display name, date/time format, and password. Select the user icon in the top navigation bar, and then select **Profile** from the drop-down menu.
+
+{{< figure src="/images/automate/chef-automate-profile.png" alt="Profile Section in Chef Automate">}}
+
+Select **Profile** to view the details with respect to your _username_. The **Details** tab lets you update your _Display Name_ and the _Time Format_, whereas the **Reset Password** tab lets you update your password.
+
+### Change a Display Name
+
+You can change the display name for your Chef Automate in the bar.
+
+{{< figure src="/images/automate/display-name-in-profile.png" alt="Display Name of Chef Automate">}}
+
+{{< note >}}
+The **Display Name** gets updated as soon as you change the name in the bar.
```

Comment:
Don't use a note for this. Put the content above the image. 

Also, I would think that the display name changes when the user hits `save`?

Reply:
Yes!
Will update both the pointers